src/FINDOLOGIC/Export/Data/Property.php

```diff
@@ -59,7 +61,16 @@ public function addValue($value, $usergroup = null)
             throw new DuplicateValueForUsergroupException($this->key, $usergroup);
         }
 
-        $this->values[$usergroup] = $value;
+        $this->values[$usergroup] = DataHelper::emptyValueCheck($value);
```

Comment:
[Suppress](https://phpmd.org/documentation/suppress-warnings.html) the Code Climate warning for using static access here, because this is a legitimate use case.

Reply:
Done